recipes/default.rb

```diff
@@ -22,3 +22,26 @@
 # Package
 package 'docker-engine'
 
+# Service configuration
+template '/lib/systemd/system/docker.service' do
+  source "docker.service"
+  mode 0644
+end
+
+# Service
+service "docker"
+
+# Configuration
+directory "/etc/docker" do
+  owner 'root'
+  group 'root'
+  mode 0700
+end
+
+# API TLS Key
+execute 'generate API tls key' do
```

Comment:
Could use the [`openssl_x509`](https://github.com/chef-cookbooks/openssl#openssl_x509) resource instead.